src/App.js

```diff
@@ -62,9 +38,28 @@ class App extends React.Component {
       row: parseInt(row)
     });
 
+    this.openModal(lastClicked);
+  }
+
+  onClickNav = (id) => {
+    var tableId = "#table-" + id
+    var table = document.querySelector(tableId);
+    window.scrollTo(0, (table.offsetTop - 260))
```

Comment:
❓ Where does this 260 come from? Should we save it as a constant somewhere if it's reused?